CHANGES

```diff
@@ -45,12 +45,17 @@
      [Paul Dale]
 
   *) Over two thousand fixes were made to the documentation, including:
-     common options (such as -rand/-writerand, TLS version control, etc)
-     were refactored and point to newly-enhanced descriptions in openssl.pod,
-     documented all missing options, added a CI build target to make
-     sure all options are documented, have style conformance (with help
-     from Richard Levitte), documented many internals, and addressed all
-     internal broken L<> references.
+     - Common options (such as -rand/-writerand, TLS version control, etc)
+       were refactored and point to newly-enhanced descriptions in openssl.pod.
+     - Added style conformance for all options (with help from Richard Levitte),
+       documented all reported missing options, added a CI build to check
+       that all options are documented and that no unimplemented options
+       are documented.
+     - Added a CI build target to make sure all options are documented and have
+       style conformance (with Richard Levitte), and added documentation for all
+       options which were missing it.
```

Comment:
Ups, the third item was an abandoned attempt of me to reformulate the second. Please flip a coin and choose your favorite version :-).

Reply:
the first one.  updated commit pushed.